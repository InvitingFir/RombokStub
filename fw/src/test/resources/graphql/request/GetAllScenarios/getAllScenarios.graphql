query getAllScenarios($uuid: String!){
  getAllScenarios(serviceUuid: $uuid){
    scenarioType
    name
    uuid
    isDefault
    variables {
      name
      value
    }
    url
  }
}