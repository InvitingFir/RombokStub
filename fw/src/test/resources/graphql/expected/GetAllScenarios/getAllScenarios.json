{
  "data": {
    "getAllScenarios": [
      {
        "scenarioType": "HTTP",
        "name": "name",
        "uuid": "5ce891dc-b61e-4e8a-b183-21b58c31ba66",
        "isDefault": false,
        "variables": [
          {
            "name": "var1",
            "value": "value"
          }
        ],
        "url": "null"
      },
      {
        "scenarioType": "HTTP",
        "name": "name",
        "uuid": "5ce891dc-b61e-4e8a-b183-21b58c31ba66",
        "isDefault": false,
        "variables": [
          {
            "name": "var1",
            "value": "value"
          }
        ],
        "url": "null"
      }
    ]
  }
}